tasks/bash.sh

```diff
@@ -140,6 +140,7 @@ munge_name() {
     arista) echo "AristaEOS" ;;
     huawei) echo "HuaweiOS" ;;
     photon) echo "PhotonOS" ;;
+    freebsd) echo "FreeBSD" ;;
```

Comment:
Hmm, my main concern with this is it could be breaking for anyone that's consuming these facts, but at least looking at Plans (which seems like the main place you would do that) it doesn't seem like [anyone is doing that](https://github.com/search?q=freebsd+path%3Aplans). I also didn't find a lot of [scripts running this task directly](https://github.com/search?q=%22bolt+task+run+facts%22&type=code). This definitely seems like a bug / miss, so I'm +1!

Reply:
A lot of things with Bolt are unfortunately not working well on FreeBSD due to Bolt modules thinking everything use AIO packaging.

I am working on improving this while working on this personal project https://github.com/smortex/dotfiles ; one step at a time :-)